cmd/generate.go

```diff
@@ -0,0 +1,45 @@
+package cmd
+
+import (
+	"dynamic-buildkite-template/generator"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	// cobra.OnInitialize()
+	generateCmd.PersistentFlags().String("trivyPlugin", "v1.18.2", "provide trivy plugin version")
```

Comment:
I would leave an empty default here, thats good get outdated quickly.

Reply:
Done...Still few other amendments are working in progress